src/cli/configure/upgrade/index.ts

```diff
@@ -0,0 +1,82 @@
+import { readdir, writeFile } from 'fs/promises';
+import path from 'path';
+
+import { gte, sort } from 'semver';
+
+import { log } from '../../../utils/logging';
+import { getConsumerManifest } from '../../../utils/manifest';
+import { getSkubaVersion } from '../../../utils/version';
+import type { SkubaPackageJson } from '../../init/writePackageJson';
+import { formatPackage } from '../processing/package';
+
+const getPatches = async (manifestVersion: string): Promise<string[]> => {
+  const patches = await readdir(path.join(__dirname, 'patches'));
+
+  // The patches are sorted by the version they were added from.
+  // Only return patches that are newer or equal to the current version.
+  return sort(patches.filter((filename) => gte(filename, manifestVersion)));
+};
+
+const fileExtensions = ['js', 'ts'];
+
+export type Patch = {
+  upgrade: () => Promise<void>;
```

Comment:
Just thinking, would a patch ever want/need to know any metadata about the current project? For example, could we introduce a patch that should only be applied to projects templated from `lambda-sqs-worker-cdk`?

Reply:
Eh we can add it down the line if we decide to :P

Reply:
Yep not for this PR, just thinking about the thinking :)